onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -75,6 +75,9 @@ def __init__(self, value: torch.Value):
         self._shape: Optional[Tuple[int | None, ...]] = None
         self._name: Optional[str] = None
 
+    def __repr__(self):
+        return f"TorchScriptTensor('{self._torch_value!r}')"
```

Comment:
Should it be composed from other properties such as shape/dtype/name other than `_torch_value`?

Reply:
sure, I was just lazy lol

Reply:
Actually, I think it's appropriate since it was initialized this way. Granted if we eval the string it wouldn't give us back the same thing.

Reply:
> since it was initialized this way

This makes sense.